src/lib/krb5/krb/preauth2.c

```diff
@@ -499,6 +499,9 @@ responder_get_answer(krb5_context context, krb5_clpreauth_rock rock,
     /* Don't let plugins get the raw password. */
     if (question && strcmp(KRB5_RESPONDER_QUESTION_PASSWORD, question) == 0)
         return NULL;
+
+    if (question == NULL)
```

Comment:
Would prefer changing the previous condition to `!question || strcmp() == 0` instead.